ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -84,10 +98,34 @@ export class ImportCommand extends IronfishCommand {
 
     process.stdin.off('data', onData)
 
+    try {
+      data = ImportCommand.bech32ToJSON(data)
+    } catch (e) {
+      CliUx.ux.info('Unable to decode bech32, assuming input is already JSON')
+    }
     return JSONUtils.parse<AccountImport>(data)
   }
 
   async importTTY(): Promise<AccountImport> {
+    const response: { decodingChoice: string } = await inquirer.prompt<{
```

Comment:
This UX isn't great, we can do better. Lets just let the user input text, try all formats, if fails then default to asking for account name and spending key.

Reply:
The reason I don't like this is most users will likely not know what either of these types are. They will just be carrying in the account info, copy pasta.

Reply:
Good point.